### DISCUSSION TITLE
Do I start worry when I get warning from React.memo?

### DISCUSSION THREAD

**Discussion by p4rtridge** [xiaolin/react-image-gallery#747] [General]
This project is great. But I have a problem when using it on NextJS version 13.4.12 and react 18.2.0. It warns me like this:

`Warning: ae: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.`

Will that be a problem in the future? If yes then how can I fix this?

**Comment by xiaolin** [comment]
This is currently a warning, so we should be fine. I will likely look into this when I get some free time to remove the warning.

**Reply by p4rtridge** [reply]
That's a good news. Thank you for the powerful package
